tests/platform_tests/link_flap/link_flap_utils.py

```diff
@@ -191,10 +191,10 @@ def check_bgp_routes(dut, start_time_ip_route_counts, ipv4=False):
         ipv4: Version of IP
     """
     if ipv4:
-        end_time_ip_route_counts = dut.shell("show ip route summary | grep Total | awk '{print $2}'")["stdout"]
+        end_time_ip_route_counts = dut.shell("vtysh -c 'show ip route summary' | grep Total | awk '{print $2}'")["stdout"]
```

Comment:
can you wrap this in a help function to call "show ip route summary" first, if not successful, call the vtysh command to return the result?

Reply:
Why do we need that? The ```show ip route``` is actually calling ```vtysh -c``` inside.

Reply:
We should use cli command whenever possible. not having the command is an image issue and it will be addressed.

Reply:
Agree. So I think we should drop this PR and let the issue to surface out since they have confirmed that this is an image issue. What do you think?

Reply:
We can keep this PR. But put the work-around at a helper function that can be updated in the future would help.

Reply:
If the workaround is added, how could we find such issue?